src/types/list.rs

```diff
@@ -424,3 +464,10 @@ mod test {
         assert_eq!(2, list.get_item(3).extract::<i32>().unwrap());
     }
 }
+
+    }
+}
+
+    create_tests!(array);
+    create_tests!(vec vec!);
```

Comment:
Not sure if this approach is desired. If so, I will also do the same in sequence.rs.

Reply:
I think this is a good idea, thanks for adding comprehensive tests!